src/components/ObsEdit/ObsEdit.js

```diff
@@ -131,10 +148,54 @@ const ObsEdit = ( ): Node => {
     setPhotoUris( uris );
   }, [currentObs] );
 
+  const addEvidence = () => {
+    bottomSheetModalRef.current?.present();
+  };
+
   if ( !currentObs ) { return null; }
 
+  const renderBackdrop = props => (
+    <BottomSheetBackdrop
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...props}
+      pressBehavior="close"
+      appearsOnIndex={0}
+      disappearsOnIndex={-1}
+    />
+  );
+
+  const onImportPhoto = async () => {
+    const response = await launchImageLibrary( {
```

Comment:
This should open the Camera Roll component that you see when you create an observation by importing photos, not `launchImageLibrary`